image/rhel/fetch-stackrox-data.sh

```diff
@@ -4,7 +4,11 @@
 
 set -euxo pipefail
 
+TARGET_DIR="${1:-/stackrox-data}"
```

Comment:
I like how you addressed it by making the command extensible and not breaking the existing caller. I would, however, ask to take it a bit further and convert the single parameter to be required instead of being optional. This is to make it explicit _at the call site_ where the data would go.

Besides `konflux.Dockerfile`, there's just one place where this `fetch-stackrox-data.sh` is called upstream. The script is not usable downstream so you don't need to worry about breaking anything there.

```suggestion
if [[ "$#" -lt "1" ]]; then
	>&2 echo "Error: please provide target directory where to store downloaded data"
	exit 6
fi

TARGET_DIR="${1}"
```

(untested)

Reply:
It works!